istioctl/pkg/writer/envoy/configdump/listener.go

```diff
@@ -104,7 +104,17 @@ func retrieveListenerType(l *listener.Listener) string {
 }
 
 func retrieveListenerAddress(l *listener.Listener) string {
-	return l.Address.GetSocketAddress().Address
+	sockAddr := l.Address.GetSocketAddress()
+	if nil != sockAddr {
+		return sockAddr.Address
+	}
+
+	pipe := l.Address.GetPipe()
```

Comment:
Thanks for the fix, It is better if you can add a test 
for this. A possible location is https://github.com/istio/istio/blob/master/istioctl/pkg/writer/envoy/configdump/listener_test.go